src/main/java/com/wedevol/xmpp/EntryPoint.java

```diff
@@ -40,5 +40,6 @@ public static void main(String[] args) {
 		CcsOutMessage message = new CcsOutMessage(toRegId, messageId, dataPayload);
 		String jsonRequest = MessageHelper.createJsonOutMessage(message);
 		ccsClient.send(jsonRequest);
+		cssClient.disconnect();
```

Comment:
but with this you are disconnecting immediately after sending the sample message. the idea is that the server keeps receiving messages